releases/development/master/extra/network-json-validator

```diff
@@ -418,16 +419,11 @@ def validate_networks databag
   end
 
   if not @networks['public'].subnet_addr_full.include?(@networks['nova_floating'].subnet_addr_full)
-    raise "'nova_floating' network must be a subnetwork of 'public' network"
-  end
-
-  if @networks['public'].use_vlan != @networks['nova_floating'].use_vlan
-    raise "The 'Use VLAN' setting (JSON attribute: 'use_vlan') of the 'nova_floating' and 'public' networks must be the same."
-  end
-
-  if @networks['public'].use_vlan
-    if @networks['public'].vlan != @networks['nova_floating'].vlan
-      raise "'nova_floating' network must have the same VLAN configuration as 'public' network"
+    if @networks['public'].use_vlan && @networks['nova_floating'].use_vlan &&
+        @networks['public'].vlan == @networks['nova_floating'].vlan
+      raise "When using the same VLAN 'nova_floating' must be a subnetwork of the 'public' network"
```

Comment:
For me we can drop also this check, by CISCO it is called router-on-stick
http://www.firewall.cx/cisco-technical-knowledgebase/cisco-routers/336-cisco-router-8021q-router-stick.html